diff --git a/node_modules/@remix-run/dev/dist/cli/commands.js b/node_modules/@remix-run/dev/dist/cli/commands.js
index 45e8490..3262313 100644
--- a/node_modules/@remix-run/dev/dist/cli/commands.js
+++ b/node_modules/@remix-run/dev/dist/cli/commands.js
@@ -291,7 +291,7 @@ async function dev(remixRoot, modeArg, portArg) {
   app.use((_, __, next) => {
     purgeAppRequireCache(config$1.serverBuildPath);
     next();
-  });
+  });  app.use("/node_modules", express.static('node_modules'));
   app.use(createApp(config$1.serverBuildPath, mode, config$1.publicPath, config$1.assetsBuildDirectory));
   let server = null;
 
diff --git a/node_modules/@remix-run/dev/dist/compiler.js b/node_modules/@remix-run/dev/dist/compiler.js
index 1b4c8e9..abbb0fa 100644
--- a/node_modules/@remix-run/dev/dist/compiler.js
+++ b/node_modules/@remix-run/dev/dist/compiler.js
@@ -221,6 +221,7 @@ async function watch(config$1, {
   let watcher = chokidar__default["default"].watch(toWatch, {
     persistent: true,
     ignoreInitial: true,
+    ignored: /app\/server\/build\//,
     awaitWriteFinish: {
       stabilityThreshold: 100,
       pollInterval: 100
