diff --git a/node_modules/ngrok/src/client.js b/node_modules/ngrok/src/client.js
index 95cd729..84432f0 100644
--- a/node_modules/ngrok/src/client.js
+++ b/node_modules/ngrok/src/client.js
@@ -29,7 +29,8 @@ class NgrokClient {
     } catch (error) {
       let clientError;
       try {
-        const response = JSON.parse(error.response.body);
+        console.error("ERROR", e)
+        const response = error.response ? JSON.parse(error.response.body) : {msg: "No error response found"};
         clientError = new NgrokClientError(
           response.msg,
           error.response,
@@ -39,7 +40,7 @@ class NgrokClient {
         clientError = new NgrokClientError(
           error.response.body,
           error.response,
-          error.response.body
+          error.response.body,
         );
       }
       throw clientError;
