diff --git a/node_modules/ngrok/src/client.js b/node_modules/ngrok/src/client.js
index 95cd729..f162b7f 100644
--- a/node_modules/ngrok/src/client.js
+++ b/node_modules/ngrok/src/client.js
@@ -29,17 +29,19 @@ class NgrokClient {
     } catch (error) {
       let clientError;
       try {
-        const response = JSON.parse(error.response.body);
+        console.error("ERROR", error)
+        const response = error.response ? JSON.parse(error.response.body) : {msg: "No error response found"};
         clientError = new NgrokClientError(
           response.msg,
           error.response,
           response
         );
       } catch (e) {
+        console.error("DOUBLE ERROR", e)
         clientError = new NgrokClientError(
-          error.response.body,
+          error.response?.body || "No error response found",
           error.response,
-          error.response.body
+          error.response?.body || "No error response found",
         );
       }
       throw clientError;
