diff --git a/node_modules/typescript/lib/tsc.js b/node_modules/typescript/lib/tsc.js
index d98c47c..3b578ce 100644
--- a/node_modules/typescript/lib/tsc.js
+++ b/node_modules/typescript/lib/tsc.js
@@ -34650,9 +34650,11 @@ var commandOptionsWithoutBuild = [
     // this option can only be specified in tsconfig.json
     // use type = object to copy the value as-is
     name: "paths",
-    type: "object",
+    type: {
+      get: (s) => JSON.parse(s)
+    },
     affectsModuleResolution: true,
-    isTSConfigOnly: true,
+    isTSConfigOnly: false,
     category: Diagnostics.Modules,
     description: Diagnostics.Specify_a_set_of_entries_that_re_map_imports_to_additional_lookup_locations,
     transpileOptionValue: void 0
